lib/src/pick.dart

```diff
@@ -8,28 +8,34 @@ import 'ls.dart';
 typedef Future<void> AfterPickedAction(BuildContext context, Directory folder);
 
 class _FolderPickerState extends State<FolderPicker> {
-  _FolderPickerState({@required this.action, @required this.rootDirectory}) {
-    _currentDirectory ??= rootDirectory;
```

Comment:
Root directory now refers to the actual root directory, whose parent can't be accessed by the user. When the state is created, it checks if the user has permissions to access the current folder's parent, if not, the current folder becomes now the root folder, preventing the user from going up on the folder tree.